src/components/TimePicker/TimePicker.driver.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  timePickerDriverFactory as coreTimePickerDriverFactory
+} from 'wix-ui-core/dist/src/components/TimePicker/TimePicker.driver';
+
+export const timePickerDriverFactory = ({element, eventTrigger}) => {
+  const coreTimePickerDriver = coreTimePickerDriverFactory({element, eventTrigger});
+
+  return {
+    ...coreTimePickerDriver,
+    getValue: () => coreTimePickerDriver.getValue().slice(0, 5),
```

Comment:
No need to slice, value is already in `hh:mm` 24-hour format

Reply:
value could have `am` or `pm` in it, so still need to extract that somehow

Reply:
value could have `am` or `pm` in it, so still need to extract that somehow